ddl/placement/bundle.go

```diff
@@ -49,6 +49,224 @@ func NewBundle(id int64) *Bundle {
 	}
 }
 
+// NewBundleFromConstraintsOptions will transform constraints options into the bundle.
+func NewBundleFromConstraintsOptions(constraints string, leaderConstraints string, learnerConstraints string, followerConstraints string, voterConstraints string, followerCount uint64, voterCount uint64, learnerCount uint64) (*Bundle, error) {
+	if voterCount+followerCount == 0 {
```

Comment:
Not support only 1 leader replica without any followers?

Reply:
Yeah, it does not make sense anyway. @morgo , what do you think about?

Reply:
Once case is that user only rely on the scalebility of TiDB, but the they will do the hot standby themselves

Reply:
I've changed the behavior.